cmd/mattermost-mattermod/migrations.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/mattermost/mattermost-mattermod/store/migrations"
+
+	_ "github.com/go-sql-driver/mysql" // Load MySQL Driver
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/mysql"
+	bindata "github.com/golang-migrate/migrate/v4/source/go_bindata"
+)
+
+func runMigrations(driverName, dataSource string, migrateVersion int) error {
```

Comment:
I see that most of this is duplicated in the store with the main difference being that there we go up only.
Would it make sense to have a more generic version over there which can be also re-used here?

Reply:
It's just creating the database and source driver which can be de-duped. And we'll have to do the error checking anyways. So it'll just replace 2 function calls with 1. Not sure if it's worth it.

Reply:
I think all the `m *Migrate` initialization part could be a separate function to be used in the two places. One will run `m.Migrate()` and the other `m.Up()`. But yes, not a big deal. It's also fine as it is.